CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+
+1.1.0-M1 / 2020-12-29
```

Comment:
Shouldn't we mention the support of inline classes and performance optimizations in the plugin?